media-gfx/openexr_viewers/files/openexr_viewers-2.2.0-add-missing-files.patch

```diff
@@ -0,0 +1,160 @@
+diff -purN a/CMakeLists.txt b/CMakeLists.txt
+--- a/CMakeLists.txt	1970-01-01 01:00:00.000000000 +0100
```

Comment:
so here you're adding the whole cmake build system ?!?

it's also missing upstream reference / PR